metadata/geoscience/soils.md

```diff
@@ -0,0 +1,73 @@
+# Title
+
+Utah Soils
+
+## ID
+
+e909493a-2ab0-465e-83c8-aebbfae4c6e8
+
+## Brief Summary
+
+Soil characteristics of Utah derived from the Soil Survey Geographic Database (SSURGO).
+
+## Summary
+
+Soil characteristics of Utah derived from the Soil Survey Geographic Database (SSURGO). This dataset represents soil map units as polygons and includes attributes such as soil type and a generalized description of soil texture.
+
+## Description
+
+### What is the dataset?
+
+Utah Soils is a subset of the more comprehensive Natural Resources Conservation Service (NRCS) [SSURGO database](https://www.nrcs.usda.gov/resources/data-and-reports/soil-survey-geographic-database-ssurgo). UGRC has modified the original source data to provide a Utah-focused soils dataset for public and private use.
+
+### What is the purpose of the dataset?
+
+This dataset provides Utah-based users with a subset of the original SSURGO database that has been modified by UGRC for simplicity and clarity. Soil type data are used in natural resource management, government planning, agriculture, and many other areas.
+
+### What does the dataset represent?
+
+Each polygon in this dataset represents a geographic area characterized by a specific soil type. Soil type descriptions can be found under the musurftexgrp field. For a more detailed description of soil types in Utah, please see the [SSURGO database](https://nrcs.app.box.com/v/soils).
+
+### How was the dataset created?
+
+The SSURGO database is a collection of feature classes, tables, and relationship classes that represent over a century's worth of soil data collected by the [National Cooperative Soil Survey](https://www.nrcs.usda.gov/about/partner-with-us/national-cooperative-soil-survey). UGRC had modified this source data to create a collection of the more widely used SSURGO soil attributes and SSURGO geometry to provide users an overview of Utah soils data.
```

Comment:
Could be good to add something like this here... UGRC's schema is based off the more commonly used fields in SSURG and a [Soil Data Access Query](https://sdmdataaccess.nrcs.usda.gov/Query.aspx) to return a dataset that does not rely on the relationship classes found in SSURGO.

Reply:
I'll add that in!